packages/manager/modules/nutanix/src/dashboard/nodes/node/interventions/module.js

```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+
+import { serverInterventions } from '@ovh-ux/manager-bm-server-components';
+
+import routing from './routing';
+import component from './component';
+
+const moduleName = 'ovhManagerNutanixNodeInterventions';
+
+angular
+  .module(moduleName, ['ui.router', serverInterventions])
```

Comment:
**question:** Since you are using `breadcrumb` into the routing part, shouldn't `ngUiRouterBreadcrumb` be injected as a module dependency here?

Reply:
I need clarification here. I don't see breadcrumb used in all modules. Its mainly used in the parent module. I have used it in the parent module of Nutanix. Since these modules (nutanix child modules) can not be used in isolation, I don't see the need for it. But if it is something a guideline or practice, I can add it.